### ISSUE TITLE
Sending request to nodeSSPI on a remote server via XHR?

### ISSUE THREAD

**Issue by BrianJVarley** [abbr/NodeSSPI#35]
@abbr I added the nodeSSPI Api call to an express server. Everything works fine on a local host request via Ajax and the user's name is returned from `req.connection.user;`

But when I deploy the express server to a remote machine on a Windows 7 box the server denies only this request calling the nodeSSPI Api - `app.get('/OOO/UserLDAP', UserLDAP.Authenticate); ` giving back a 401 error-

> Failed to load resource: the server responded with a status of 401 (Unauthorized)

All other GET requests are fine on the server with results returned as expected.

Do I need to pass basic auth credentials when calling this Api on a remote server? 
(Or can the required info be retrieved from the API without passing the hard coded credentials in the request header.)

This is a gist of my express server's setup as is:

https://jsfiddle.net/brianVarley/tf1puukp/

**Comment by zcmack** [comment]
i have the same issue but both of my machines are on the same domain. requests to localhost authenticate as expected, but remote requests are prompted for basic auth and are rejected regardless if a correct password is supplied.

**Comment by BrianJVarley** [comment]
@zcmack  Yes same today I changed my solution to setCredentials=true in the XHR request. Which in turn prompts for basic auth when I send a request to the remote express server. Supplying the correct credentials to the basic auth pop up are rejected. Thus giving back the 403.

In contrast with that when I send the request to the server run on local the basic auth popup accepts my credentials and returns expected Json result.


Hoping to hear something back on here as to why. :)

**Comment by abbr** [comment]
Is primary web page's host name same as xhr's?
